LondonBritishLibrary/add/BLadd24994.xml

```diff
@@ -5,7 +5,7 @@ schematypens="http://relaxng.org/ns/structure/1.0"
     <teiHeader xml:base="https://betamasaheft.eu/">
         <fileDesc>
             <titleStmt>
-                <title/>
+                <title>Apocalyptic Beta ʾƎsrāʾel texts, Canticles</title>
```

Comment:
Sorry, I miss Canticles, what is this?

Reply:
Not sure I understand what you mean :) The Biblical canticles, starting from f. 15v here